crates/kitsune_p2p/kitsune_p2p/tests/sharding.rs

```diff
@@ -260,8 +305,16 @@ async fn gossip_to_basis_from_inside() {
     let space = Arc::new(fixt!(KitsuneSpace));
 
     let tuner = |mut params: tuning_params_struct::KitsuneP2pTuningParams| {
-        params.gossip_arc_clamping = "none".to_string();
-        params.gossip_dynamic_arcs = false; // Don't update the arcs dynamically, use the initial value
+        params.gossip_arc_clamping = if cfg!(feature = "unstable-sharding") {
+            "none".to_string()
+        } else {
+            "full".to_string()
+        };
```

Comment:
This is a nitpick... but `cfg!` just compiles in a bool and forces the optimizer to delete the other branch. It's slightly more efficient to use the cfg decorator.
```suggestion
        #[cfg(feature = "unstable-sharding")]
        {
            params.gossip_arc_clamping = "none".to_string();
        }
        #[cfg(not(feature = "unstable-sharding"))]
        {        
            params.gossip_arc_clamping = "full".to_string();
        }
```

Reply:
Or use `cfg_if!` for the same thing